### PR TITLE
prosthetic-hand blog post

### PR DESCRIPTION
**Pull request by IvanSanchez**

### CODE REVIEW

File: _posts/2016-03-20-debugging-touch-interactions.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+description: To debug Leaflet, sometimes you need to create a new tool.
+author: Iván Sánchez
+authorsite: http://ivan.sanchezortega.es
+---
+
+
+Most of the time, fixing bugs in the Leaflet code is a breeze. The code is simple, easy to read (for the most part) and well structured. Code conventions and unit tests make it easy for newcomers to try some modifications to the core code. During the past few months we've sent a few simple bug reports to the folks at [Your First PR](https://yourfirstpr.github.io/) - we love to see first-timers contributing fixes to Leaflet!
+
+
+Some of the difficulties of maintaining/developing a javascript library life Leaflet is making sure that everything works on every major browser out there. A technique that works on Firefox on a Ubuntu desktop might result in glitches in Safari in a Macbook; something that works for Edge in Windows 10 might break completely in Chrome in Android.
```
Comment (jieter) [inline _posts/2016-03-20-debugging-touch-interactions.md]: ...safari in a Macbook... Did you mean 'on a Macbook?

Comment (IvanSanchez) [inline _posts/2016-03-20-debugging-touch-interactions.md]: @jieter No caffeine in monday equals no good word togethering. :-)

File: _posts/2016-03-20-debugging-touch-interactions.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+Debugging touch interactions is particularly tricky. Sometimes the conditions to reproduce a touch-interaction bug is simple (double-tap the touchscreen in the same spot), but sometimes they are more specific. In [#3798](https://github.com/Leaflet/Leaflet/issues/3798) and [#3814](https://github.com/Leaflet/Leaflet/issues/3814) the conditions are "drag whith one finger, then put down another finger and pinch", and in [#3530](https://github.com/Leaflet/Leaflet/issues/3530) it's "pinch in until `maxZoom` is reached, then do a two-finger drag".
+
+The problem with this kind of bugs is that they're **frustating** and **time-consuming** to reproduce under controlled conditions. Imaging having a code editor and a browser debugger when at the same time using two hands to perform a very specific touch gesture while watching the debugger. Then you want to inspect a variable in the debugger but you cannot move your fingers even a pixel because that will run more code and change the state.
```
Comment (jieter) [inline _posts/2016-03-20-debugging-touch-interactions.md]: Imaging -> Imagine

**Comment by mourner** [comment]
:+1:

**Pull request merged**